libvirt/provider.go

```diff
@@ -30,7 +30,7 @@ func Provider() terraform.ResourceProvider {
 
 func providerConfigure(d *schema.ResourceData) (interface{}, error) {
 	config := Config{
-		Uri: d.Get("uri").(string),
+		URI: d.Get("URI").(string),
```

Comment:
Same here, let's leave that to `uri` otherwise all the existing terraform files will be broken.